HeterogeneousCore/SonicTriton/interface/TritonClient.h

```diff
@@ -0,0 +1,93 @@
+#ifndef HeterogeneousCore_SonicTriton_TritonClient
+#define HeterogeneousCore_SonicTriton_TritonClient
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "HeterogeneousCore/SonicCore/interface/SonicClientSync.h"
+#include "HeterogeneousCore/SonicCore/interface/SonicClientPseudoAsync.h"
+#include "HeterogeneousCore/SonicCore/interface/SonicClientAsync.h"
+
+#include <vector>
+#include <string>
+#include <exception>
+
+#include "request_grpc.h"
+
+namespace ni = nvidia::inferenceserver;
+namespace nic = ni::client;

```

Comment:
Quick search suggests that namespace aliases in headers can be a bad idea (because of risks in name collisions, in global namespace in this case).

Reply:
Agreed...